settings.gradle

```diff
@@ -13,3 +13,4 @@ include ':shark-graph'
 include ':shark-hprof'
 include ':shark-log'
 include ':shark-test'
+include ':leakcanary-android-proguard'
```

Comment:
I'm not seeing the config needed to publish to maven central with upload jars, shouldn't there be a properties file?

Reply:
Yes, like I mentioned in general discussion - I intentionally didn't add the config because I had some doubts about it.

Reply:
Maybe users would like a name that reflect the fact that it is a gradle plugin.

Reply:
Makes sense, what do you think @pyricau ?

Reply:
I like it!

Now, the difficult conversation: proguard is being replaced by R8. I have no idea how we'd support R8 here, but we'd probably want to build that in the same plugin?

Reply:
It already does support R8. The only difference is the task name: `transformClassesAndResourcesWithR8*` vs `transformClassesAndResourcesWithProguard*`. But as @JakeWharton  noticed, maybe we should rename the whole thing to `leakcanary-android-minification-plugin` or something like that? On the other hand we still have `proguardFiles getDefaultProguardFile('proguard-android-optimize.txt')` in our build.gradle files even though R8 is used under the hoods... 🤔

Reply:
`leakcanary-android-minification-plugin` looks good to me. It's a bit absconse but that's the proper name of the operation..

Reply:
Isn't it a bit misleading though? Someone may think that this plugin performs minification.

Reply:
I think `deobfuscation` better describes what it actually does